packages/node_modules/@ciscospark/plugin-conversation/src/share-activity.js

```diff
@@ -333,9 +333,8 @@ ShareActivity.create = function create(conversation, object, spark) {
     parent: spark
   });
 
-  if (files) {
-    files.forEach((file) => share.add(file));
-  }
+  files = files.items || files;
+  files.items.forEach((file) => share.add(file));
```

Comment:
files can't have an `items` property at this point.
you still need to confirm files exists before referencing its properties.